Trivia_Trainer.html

```diff
@@ -87,33 +97,40 @@ <h3>Players</h3>
 
             <div class = "row">
                 <div class = "col-md-3_1">
-                    <div class="col-md-9_1">
-                        <input type="text" class="form-control" placeholder = "Player 1 Answer Choice" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-default">
-                    </div>
+                    
+
+
+
+
+
+                <div class="col-md-9_1">
+                    <input type="text" class="form-control" placeholder = "Player 1 Answer Choice" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-default">
                 </div>
             </div>
+        </div>
 
-            <div class = "row">
-                <div class = "col-md-3_1">
-                    <div class="col-md-9_1">
-                        <input type="text" class="form-control" placeholder = "Player 2 Answer Choice" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-default">
-                    </div>
+        <div class = "row">
+            <div class = "col-md-3_1">
```

Comment:
I'm not sure about these class names. They resemble boot-strap responsive columns, but with the _1 suffix they will not function responsively, which is confusing.